spec/factories/holdings.rb

```diff
@@ -21,12 +21,12 @@
 
     factory :sudoc_holding do
       call_number { 'I 19.76:98-600-B' }
-      scheme { 'SUDOC' }
+      scheme { 'superintendent' }
```

Comment:
Should we use plausible FOLIO values (from https://github.com/sul-dlss/folio_migration/blob/63166e3163bcee0562dd0c9b96d313e1adf968ac/mapping_files/call_number_type_mapping.tsv#L6)?

```suggestion
      scheme { 'Superintendent of Documents classification' }
```